cfe_internal/enterprise/federation/federation.cf

```diff
@@ -154,6 +154,10 @@ bundle agent transport_user
           "$(ssh_config)"
         };
       "ssh_home_cil_file" string => "$(cfengine_enterprise_federation:config.federation_dir)/cftransport_ssh_home_t.cil";
+      "ssh_home_te_file" string => "$(cfengine_enterprise_federation:config.federation_dir)/cftransport_ssh_home.te";
```

Comment:
Should we scope these variables to their specific context? It's kind of annoying that for each promise type, even if the bundle is targeting something specific, you have to be very explicit in the desired context, or it's defined in many places.